websites/K/Kemono.Party/presence.ts

```diff
@@ -0,0 +1,79 @@
+const presence = new Presence({
+		clientId: "1067711765770682388",
+	}),
+	browsingTimestamp = Math.floor(Date.now() / 1000);
+
+presence.on("UpdateData", async () => {
+	let presenceData: PresenceData = {
+		largeImageKey: "https://i.imgur.com/mwpEkf3.png",
+		startTimestamp: browsingTimestamp,
+	};
+
+	const service: Record<string, PresenceData> = {
+			"Patreon ": { state: "Patreon" },
+			"Pixiv Fanbox": { state: "Pixiv Fanbox" },
+			"Gumroad ": { state: "Gumroad" },
+			"Subscribestar ": { state: "SubscribeStar" },
+			"Dlsite ": { state: "DLsite" },
+			"Discord ": { state: "Discord" },
+			"Fantia ": { state: "Fantia" },
+			"Boosty ": { state: "Boosty" },
+			"Aftian ": { state: "Aftian" },
+		},
+		pages: Record<string, PresenceData> = {
+			"/": { details: "Viewing Home Page" },
+			"/importer": { details: "importing from Paysite" },
+			"/importer/tutorial": { details: "Reading FAQ" },
+			"/account": { details: "Checking out their account" },
+			"/account/keys": { details: "Checking out their keys" },
+			"/posts": { details: "Browsing through posts" },
+			"/artists": { details: "Browsing through artists" },
+			"/artists/updated": { details: "Browsing through updated artists" },
+			"/favorites": { details: "Checking out their favorites" },
+			"/dmca": { details: "Reading DMCA notice" },
+		};
+
+	for (const [path, data] of Object.entries(pages))
+		if (location.pathname === path) presenceData = { ...presenceData, ...data };
```

Comment:
`document.location` and `location` are used often in this presence. Conisder destructuring `document.location` for consistency and simplification.